cmd/network-passt-binding/README.md

```diff
@@ -0,0 +1,50 @@
+# KubeVirt Network Passt Binding Plugin
+
+## Summary
+
+Passt network binding plugin configures VMs Passt interface using Kubevirts hook sidecar interface.
+
+It will be used by Kubevirt to offload Passt networking configuration.
+
+> _NOTE_:
+> Passt network binding is supported for pod network interfaces only.
+
+# How to use
+
+Register the `passt` binding plugin with its sidecar image:
+
+```yaml
+apiVersion: kubevirt.io/v1
+kind: KubeVirt
+metadata:
+  name: kubevirt
+  namespace: kubevirt
+spec:
+  configuration:
+    network:
+      binding:
+        passt:
+          sidecarImage: registry:5000/kubevirt/network-passt-binding:devel
```

Comment:
```suggestion
          sidecarImage: quay.io/kubevirt/network-passt-binding:devel
```

We should make it work for people on arbitrary clusters. This is usage documentation, not a development guide.

Reply:
Since this image is not published yet, how about removing the registry part? 
`kubevirt/network-passt-binding`

Reply:
This should have been changed to reflect the correct path after this gets merged.
Even if this will be missing, it will eventually be there.

Reply:
The plugin image will be created as part of Kuberivrt nightly build and on next release.
Where the plugin tag from Kubevirt nightly builds will be in form of `quay.io/kubevirt/network-passt-binding:20231017_a8065ea94` 
The plugin tag on next release would be `quay.io/kubevirt/network-passt-binding:v1.2` 

Having the tag `devel` will point to an image that doenst exist, same goes for `latest` or not specifying a tag (fallback to latest).
As soon as we have a consumable image tag, I will post a PR to updated this README.